ui/App.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react'
+import { useDropzone } from 'react-dropzone'
+import Select from 'react-select'
+
+const options = [
+  { value: 'mainnet', label: 'Ethereum Mainnet' },
```

Comment:
Moving this to constants file (or json) could be better so contents can be updated more easily in the future. It's nothing crucial, just a style of coding.

Reply:
Agree the practice is a good one. My instinct is that adding another file is overkill for now, since there's no foreseeable reuse - but I think a good middle ground is to bring it inside of the component to clearly convey it is only used there.

Reply:
Okay, sounds good